pretyping/cbv.ml

```diff
@@ -325,7 +375,7 @@ and cbv_stack_value info env = function
  * normalize (even under binders) the applied terms and we build the
  * final term
  *)
-let rec apply_stack info t = function
+and apply_stack info t = function
   | TOP -> t
```

Comment:
Probably the unnecessary residual of a mutual recursivity whose need finally disappeared...